README.md

```diff
@@ -1,2 +1,151 @@
-# Leader-board
-a simple leader board which works with API
+<a name="readme-top"></a>
+
+<div>
+
+  <h1><b>Leader board</b></h1><br/><br/>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Leader board <a name="about-project"></a>
+
+**Leader board**
```

Comment:
- **[OPTIONAL]** Kindly write about what your app is for. It helps people to use your app even if they are not programmers. To have a professional README file.

Reply:
done